src/lxc/parse.c

```diff
@@ -78,7 +78,7 @@ int lxc_char_left_gc(const char *buffer, size_t len)
 
 int lxc_char_right_gc(const char *buffer, size_t len)
 {
-	int i;
+	size_t i;
```

Comment:
That should either remain `int i` or you turn `size_t len` and `int i` into `ssize_t`.